packages/nextjs/app/cow/_components/PoolCreation.tsx

```diff
@@ -19,7 +19,6 @@ export const PoolCreation = ({ state, clearState }: ManagePoolCreationProps) =>
   const token2RawAmount = parseUnits(state.token2Amount, state.token2.decimals);
   const [currentStep, setCurrentStep] = useState(1);
   const [userPoolAddress, setUserPoolAddress] = useState<Address>();
-  const [poolFinalized, setPoolFinalized] = useState<boolean>(false);
```

Comment:
Don't think we need separate state for this since same as `currentStep === 6` which now happens `onSuccess` of `finalizePool`